internal/pkg/events/incoming/ceilometer.go

```diff
@@ -0,0 +1,142 @@
+package incoming
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+)
+
+//ceilomterGenericIndex value represents ElasticSearch index name for data from which it
+// is not possible to clearly construct indexs name
+const ceilometerGenericIndex = "ceilometer_generic"
+
+var (
+	ceilometerAlertSeverity = map[string]string{
+		"audit":    "info",
+		"info":     "info",
+		"warn":     "warning",
+		"warning":  "warning",
+		"critical": "critical",
+		"error":    "critical",
+		"AUDIT":    "info",
```

Comment:
Also need a `SAMPLE` priority type I think? I saw it in the data captured as `priority: SAMPLE`.

Reply:
I think you are right. It does set priorities to an to `unknownSeverity` when the value is not included in the map. Maybe only the important ones are included here?

Reply:
What I'm not sure about is why the priority is set to SAMPLE at all.